### DISCUSSION TITLE
Keeping articles from list from showing in "recent" posts section

### DISCUSSION THREAD

**Discussion by mackiser** [jpanther/congo#160] [Q&A] [answered]
Hi James!

I wanted to see if there was an easy way to stop the items in a list/project from showing in the "recent posts" section on the homepage.

I currently have a list to keep track of my Wordle attempts and I have it separated by month but I would like to keep those out of the "Recent posts" section on the homepage and only display items from the /posts/ folder. I hope that makes sense!

![image](https://user-images.githubusercontent.com/2199450/160717066-12ac7a6c-03f7-4dbd-b838-6f848cad4959.png)
![image](https://user-images.githubusercontent.com/2199450/160717343-21c16e78-fd23-4b53-93f6-06e41908f85a.png)
![image](https://user-images.githubusercontent.com/2199450/160717372-a9164d37-91a7-4994-9e1a-18ee99c514ca.png)

**Comment by cbelena** [comment, marked as answer]
Hi, I think what you are looking for is this:

> The articles listed in this section are derived from the `homepage.mainSections` setting which allows for whatever content types you are using on your website. For instance, if you had content sections for posts and projects you could set this setting to `["posts", "projects"]` and all the articles in these two sections would be used to populate the recent list.

This is extracted [from the docs](https://jpanther.github.io/congo/docs/homepage-layout/). You have to set that setting in `params.conf` file. You can view an example [here](https://github.com/jpanther/congo/blob/dev/config/_default/params.toml).

By the way, @jpanther, I think it should be `mainSections` instead of `homepage.mainSections` in that paragraph above.

Hope it helps! 😀

**Reply by jpanther** [reply]
Ah yes, thank you for pointing out that issue - it should just be `mainSections`. I'll fix up the docs for the next release.

**Comment by mackiser** [comment]
@cbelena That worked! Thanks for the info. Looks like I glossed over that section in the docs by mistake. 😄
